include/directx/d3dx12_core.h

```diff
@@ -721,7 +721,7 @@ struct CD3DX12_RASTERIZER_DESC2 : public D3D12_RASTERIZER_DESC2
     }
     operator D3D12_RASTERIZER_DESC() const noexcept
     {
-        return (D3D12_RASTERIZER_DESC)CD3DX12_RASTERIZER_DESC1((D3D12_RASTERIZER_DESC1)*this);
+        return static_cast<D3D12_RASTERIZER_DESC>(CD3DX12_RASTERIZER_DESC1(static_cast<D3D12_RASTERIZER_DESC1>(*this)));
```

Comment:
I probably would've chosen something like:
```suggestion
        D3D12_RASTERIZER_DESC1 raw_desc1(*this);
        CD3DX12_RASTERIZER_DESC desc(raw_desc1);
        return desc;
```

I'm fine with what's here though, it obviously works and doesn't change the semantics at all.

Reply:
I was sticking with the cast form since that's what the origina author used :)